File: Backend/TrainingZone/TrainingZone/Models/DataBase/Schedule.cs
namespace TrainingZone.Models.DataBase
{
    public class Schedule
    {
        public int Id { get; set; }
        public int ClassId { get; set; }
        public Class Class { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public int MaxCapacity { get; set; }
        public decimal Price { get; set; }
        public DateTime StartDateTime { get; set; }
        public DateTime EndDateTime { get; set; }
    }
}

File: Frontend/training-zone/src/models/schedule.ts
import { ClassType } from "./enums/class-type.ts";
import { Trainer } from "./trainer.ts";

export interface Schedule {
  Id: number;
  ClassId: number;
  ClassType: ClassType;
  Description: string;
  Trainer?: Trainer;
  MaxCapacity: number;
  Price: number;
  StartDateTime: Date;
  EndDateTime: Date;
}
